app/src/shared/modules/user/actions.js

```diff
@@ -173,12 +173,15 @@ export const updateCurrentUserImage = (image: ?string) => (dispatch: Function, g
         })
 }
 
-export const saveCurrentUser = (user: User) => (dispatch: Function) => {
+export const saveCurrentUser = () => (dispatch: Function, getState: Function) => {
```

Comment:
How about we change this line to
```js
export const saveCurrentUser = () => async (dispatch: Function, getState: Function) => {
```
? This will make it always return a `Promise` and let us remove that extra `try…catch` block in [`ProfilePage#onSave`](https://github.com/streamr-dev/streamr-platform/pull/202/files#diff-006a5869adcdc369f42f1eeaeb3d220cR51). Thoughts?

Reply:
Yea, it's good idea and cleaner solution 👍